sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapWithNamedConventionsExamples.cs

```diff
@@ -20,15 +20,32 @@ protected override void CreateAndUpsertMarketDataToLusid(string scope, ModelSele
             // For LUSID to pick up these quotes, we have added a RIC rule to the recipe (see BuildRecipeRequest in TestDataUtilities.cs) 
             // The RIC rule has a large quote interval, this means that we can use one reset quote for all the resets.
             // For accurate pricing, one would want to upsert a quote per reset. 
-            var quoteRequest = TestDataUtilities.BuildQuoteRequest("USD6M", QuoteSeriesId.InstrumentIdTypeEnum.ClientInternal, 0.05m, "USD", TestDataUtilities.EffectiveAt);
+            
+            var d =  new DateTimeOffset(2019, 12, 31, 0, 0, 0, TimeSpan.Zero);
```

Comment:
"effectiveDate"? why this special one, and not a generic one?

Reply:
Will use the generic resetDate.